packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,60 @@
+
+.navMenuContainer {
+    display: flex;
+    flex-direction: column;
+    padding: 36px var(--space-base) 0 var(--space-base);
+    height: calc(100dvh - 36px);
```

Comment:
If we set `box-sizing: border-box`, the `calc` here can go away and `100dvh` will work

Reply:
![Screenshot 2024-12-11 at 1 10 28 PM](https://github.com/user-attachments/assets/23653700-ebbd-4db6-b924-47d8c565aece)
![Screenshot 2024-12-11 at 1 10 35 PM](https://github.com/user-attachments/assets/d0c3ae77-50e7-49d6-b9cc-71ca314f3d61)

This is making it skinnier. Would it be easier to make a ticket and assign it to you to go through the css?